src/Controllers/CMSExternalLinksController.php

```diff
@@ -24,6 +26,9 @@ class CMSExternalLinksController extends Controller
      */
     public function getJobStatus()
     {
+        if (!Permission::checkMember(Security::getCurrentUser(), ['CMS_ACCESS_CMSMain'])) {
```

Comment:
```suggestion
        if (!Permission::check('CMS_ACCESS_CMSMain')) {
```
Sorry, should have noticed this the first time around

Reply:
Updated